Framework/DataHandling/src/LoadMuonNexus1.cpp

```diff
@@ -631,9 +635,9 @@ void LoadMuonNexus1::loadData(size_t hist, specnum_t &i, specnum_t specNo, MuonN
   auto timeChannels = new float[lengthIn + 1]();
```

Comment:
I'm being picky now I spot things. We're aiming to drive all uses of bare `new`/`delete` as close to zero as is feasible. Can we swap this for `std::vector<float> timeChannels(lengthIn+1)`?

You can use the `.data()` method on vector in the `nxload` command below to get the bare array.

Reply:
In the next line where it is used in BinEdges, should .data() be used here as well or .begin()?